.github/workflows/release.yml

```diff
@@ -0,0 +1,53 @@
+name: Create Release
+
+on:
+  push:
+    # Sequence of  patterns matched against refs/tags
+    tags:
+      - 'v*' # Push events to  matching v*, i.e. v1.0, v20.15.10
```

Comment:
The tag has to match our requirements.

Reply:
@asa1997 
Based on requirement the tag name will be like, 1.2.24-bes0.1-cve-2017-18349

Will change it.